offboarding.qmd

```diff
@@ -10,46 +10,49 @@ The convention of giving two weeks notice is *not enough time to wrap up*.
 With more time, the group can help with next steps including finding new positions and creating an exit plan.
 Further, it typically takes a month or more to effectively document and transfer knowledge of what you are working on.
 
-### Resources
-
--   University of Arizona's [official "leaving" page](https://hr.arizona.edu/employees-affiliates/leaving-ua)
-
-### Logistics
-
--   Make sure Kristina knows what your last day will be
--   Return university-owned equipment (computers, books, etc.) to Kristina
--   Create a list of any accounts or permissions that will need to be transferred, and start transferring them.
--   Turn off access to second floor BSRL on Cat Card
--   Leave contact information (e.g., personal email address) in case we need to reach you.
-
-Websites and passwords (Kristina)
-
--   change permissions as appropriate in group organizations (optional, depending on planned future work)
-
-    -   github organizations <!--# probably needs updating -->
-
-        -   az-digitalag
-        -   agpipeline
-        -   terraref
-        -   pecanproject
-        -   sentinel-detection
-        -   genophenoenvo
-
-    -   Google Drive "Group" and other project-specific folders
-
-    -   Openstack CALS group <!--# probably not relevant anymore -->
-
--   Make sure any accounts and websites are transitioned
-
-    -   E.g. dockerhub, google drive
-
--   Remove user accounts and home directories from machines <!--# probably not relevant anymore -->
-
-    -   `sudo userdel $name && sudo rm -r /home/$name`
-
--   Remove from UA HPC group
-
-    -   go to <https://portal.hpc.arizona.edu> \> Manage Groups \> kristinariemer
+### Before final day
+
+In approximately chronological order
+
+-   Inform Kristina about what your last day will be
+-   Review University of Arizona's [official "leaving" page](https://hr.arizona.edu/employees-affiliates/leaving-ua)
+-   Write resignation letter for UA
+    -   Kristina will send this and final date to HR & ALVSCE business office
+-   Review two university checklists in Google Drive [offboarding folder](https://drive.google.com/drive/folders/1t__w3Lgrf7uju6SxXb0pGkfNE0LlTdFQ?usp=drive_link) and complete relevant tasks
```

Comment:
Do we want to be responsible for keeping these up-to-date, or should we instead link to these on the HR page (if possible)?

Reply:
Because these checklists probably won't change substantially over time, while links to the HR website pages probably will, I think I will leave it this way for the time being.